docs/how-to-provision-using-dps.md

```diff
@@ -272,36 +290,42 @@ You should see the device being registered and connected to IoT Hub, and then st
 
 ### Validate
 
-The device will be automatically registered in Azure Digital Twins. Using the following command to find the Twin of the device in the Azure Digital Twins instance you created.
+The device will be automatically registered in Azure Digital Twins. Using the following command to find the twin of the device in the Azure Digital Twins instance you created.
 
 ```azurecli-interactive
 az dt twin show -n <Digital Twins instance name> --twin-id <Device Registration ID>"
 ```
 
```

Comment:
It'd be great to show a screenshot of output here.

Reply:
Agree. Will add a screenshot.